recipe/build-libadios.sh

```diff
@@ -53,31 +53,37 @@ cmake               \
     -S "${SRC_DIR}" \
     -B build        \
     -GNinja         \
-    -DCMAKE_BUILD_TYPE=Release                \
-    -DBUILD_SHARED_LIBS=ON                    \
-    -DBUILD_TESTING=${RUN_TESTS}              \
+    -DADIOS2_BUILD_EXAMPLES=OFF               \
+    -DADIOS2_HAVE_ZFP_CUDA=OFF                \
+    -DADIOS2_INSTALL_GENERATE_CONFIG=OFF      \
+    -DADIOS2_RUN_INSTALL_TEST=OFF             \
```

Comment:
Should this be removed?
```suggestion
```

Reply:
No, this is intended so that we do not generate the adios2-config script (not to be confused with the adios2Config.cmake file). This script was a drop-in replacement of pkg-config, however, it has been proved to be very hard to maintain since it relies on reverse engineering a dummy cmake build. This reverse engineering process is broken in newer cmakes since it changes how it generate its makefiles.

As written in the desc, I am of the opinion of dropping this in the Conda build since I suspect that we do not have users that use Makefile/autotools to build against adios2 their app.

What do you think?

Reply:
I mean the `-DADIOS2_RUN_INSTALL_TEST=OFF` flag appears twice now :) Once can be removed